app/_src/mesh/features/provenance-verification-images.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Verify Build Provenance for Signed Kong Mesh Images
+badge: enterprise
+---
+
+Starting with 2.8.0, {{site.mesh_product_name}} produces build provenance for docker container images, which can be verified using `cosign` / `slsa-verifier` with attestations published to a Docker Hub repository.
+
+This guide provides steps to verify build provenance for signed {{site.mesh_product_name}} Docker container images using:
+
+* An example, to verify an image provenance leveraging any optional annotations for increased trust
+
+For the example, you will need a Docker manifest digest, a GitHub repo name, as well as any of the optional annotations you wish to verify:
+
+{:.important .no-icon}
+> The Docker manifest digest is required for build provenance verification. The manifest digest can be different from the platform specific image digest for a specific distribution.
+
+| Shorthand | Description | Example Value |
+|---|---|---|
+| `<repo>` | GitHub repository | `kong-mesh` |
+| `<workflow name>` | GitHub workflow name | `build-test-distribute` |
+| `<workflow trigger>` | Github workflow trigger name | `push` |
+| `<version>` | Artifact version to download | `{{page.version}}` |
+
+Because Kong uses GitHub Actions to build and release, Kong also uses GitHub's OIDC identity to generate build provenance for container images, which is why many of these details are GitHub-related.
```

Comment:
Same comment here about maybe adding these to the prereqs if customers have to complete this before doing the rest of the steps.

Reply:
This table is pre-populated static values when there were example templates. This is no longer needed and the examples themselves have these values that customers can copy and run directly

Reply:
I have removed these lines.